add INC_FORCE_DEVICE introduction (#1988)
intel/neural-compressor: docs/source/3x/PyTorch.md

    quant_config.set_local(".*mlp.*", RTNConfig(bits=8))  # For layers with "mlp" in their names, set bits=8
    quant_config.set_local("Conv1d", RTNConfig(dtype="fp32"))  # For Conv1d layers, do not quantize them.
    ```

3. How to specify an accelerator?

    > Neural Compressor provides automatic accelerator detection, including HPU, XPU, CUDA, and CPU.

>     > The automatically detected accelerator may not be suitable for some special cases, such as poor performance, memory limitations. In such situations, users can override the detected accelerator by setting the environment variable `INC_FORCE_DEVICE`.

Reviewer: is it better to set_device in Python as INC API?

Reviewer: Or use env variable INC_TARGET_DEVICE

Author: Hi @hshen14 sorry for the late reply. This argument will be used when `import neural_compressor`, so python API is too late to update the target device.  I agree to use `INC_TARGET_DEVICE` as you suggested, thanks a lot.

>     > The automatically detected accelerator may not be suitable for some special cases, such as poor performance, memory limitations. Therefore, Neural Compressor extends an environment variable `INC_FORCE_DEVICE` to override the automatically detected device settings for convenience.

Reviewer 2: ```suggestion
    > The automatically detected accelerator may not be suitable for some special cases, such as poor performance, memory limitations. In such situations, users can override the detected accelerator by setting the environment variable `INC_FORCE_DEVICE`.
```
